app/models/order.rb

```diff
@@ -1,10 +1,15 @@
 class Order <ApplicationRecord
   validates_presence_of :name, :address, :city, :state, :zip
 
+  belongs_to :user
   has_many :item_orders
   has_many :items, through: :item_orders
 
   def grandtotal
     item_orders.sum('price * quantity')
   end
+
+  def status
+    "Pending"
+  end
```

Comment:
Noted that this needs to be updated at a later point for actual functionality. Current user stories didn't provide any further insight into calculation of status.